devops/scripts/benchmarks/main.py

```diff
@@ -468,6 +468,10 @@ def validate_and_parse_env_args(env_args):
         type=int,
         help="Number of build jobs to run simultaneously",
         default=options.build_jobs,
+        "--hip-arch",
```

Comment:
I think something went wrong with rebase...? it's missing `parser.add_argument(` now

Reply:
Yup, I missed it up a little, fixed now 😅